### PR TITLE
EN-3344: integration test for bls multi sign

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

File: integrationTests/multisig/blsMultisig_test.go [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+func TestMultiSig_Bls(t *testing.T) {
+	t.Parallel()
+
+	consensusGroupSize := uint16(6)
+	numOfSigners := uint16(6)
+	message := "message"
+
+	bitmapSize := consensusGroupSize/8 + 1
+	// set bitmap to select all 21 members
```
Comment (AdoAdoAdo) [inline integrationTests/multisig/blsMultisig_test.go]: comment not matching - 21?

Comment (bogdan-rosianu) [reply integrationTests/multisig/blsMultisig_test.go]: removed 21

File: integrationTests/multisig/blsMultisig_test.go [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+	var err error
+	var muSig crypto.MultiSigner
+
+	for i := uint16(0); i < grSize; i++ {
+		muSig, err = multiSigners[i].Create(pubKeys, i)
+		if err != nil {
+			return err
+		}
+
+		muSigBn, ok := muSig.(crypto.MultiSigner)
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply integrationTests/multisig/blsMultisig_test.go]: done

File: integrationTests/multisig/blsMultisig_test.go [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+	return nil
+}
+
+func verifySigAllSignersBls(
+	multiSigners []crypto.MultiSigner,
+	message []byte,
+	signature []byte,
+	pubKeys []string,
+	bitmap []byte,
+	grSize uint16) error {
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply integrationTests/multisig/blsMultisig_test.go]: done

File: integrationTests/multisig/blsMultisig_test.go [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+	llSigner := &llsig.KyberMultiSignerBLS{}
+
+	for i := uint16(0); i < nbSigs; i++ {
+		multiSigners[i], _ = multisig.NewBLSMultisig(llSigner, hasher, pubKeysStr, privKeys[i], kg, i)
+	}
+
+	return pubKeysStr, multiSigners
+}
+
+func createSignaturesShares(nbSigs uint16, multiSigners []crypto.MultiSigner, message []byte) [][]byte {
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply integrationTests/multisig/blsMultisig_test.go]: renamed to numOfSigners

File: integrationTests/multisig/blsMultisig_test.go [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+	"github.com/ElrondNetwork/elrond-go/crypto/signing/kyber"
+	llsig "github.com/ElrondNetwork/elrond-go/crypto/signing/kyber/multisig"
+	"github.com/ElrondNetwork/elrond-go/crypto/signing/multisig"
+	"github.com/ElrondNetwork/elrond-go/hashing"
+	"github.com/ElrondNetwork/elrond-go/hashing/blake2b"
+	"github.com/stretchr/testify/assert"
+)
+
+func createMultiSignersBls(
+	nbSigs uint16,
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply integrationTests/multisig/blsMultisig_test.go]: numOfSigners

**Pull request merged**